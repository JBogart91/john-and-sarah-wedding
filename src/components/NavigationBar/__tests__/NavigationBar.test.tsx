import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import { NavigationBar } from '..'

describe('NavigationBar', () => {
  it('renders correctly', () => {
    const result = render(<NavigationBar />)
    expect(result).toMatchSnapshot()
  })
})
