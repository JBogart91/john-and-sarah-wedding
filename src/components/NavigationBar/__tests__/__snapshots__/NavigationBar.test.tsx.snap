// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="w-full flex py-4 justify-between items-center navbar border-b-4 border-solid"
      >
        <ul
          class="list-none sm:flex hidden justify-end items-center flex-1"
        >
          <li
            class="cursor-pointer text-base
              border-b-2 border-solid border-black
              mr-6"
          >
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="cursor-pointer text-base
              
              mr-6"
          >
            <a
              href="/our-story"
            >
              Our Story
            </a>
          </li>
          <li
            class="cursor-pointer text-base
              
              mr-6"
          >
            <a
              href="/rsvp"
            >
              RSVP
            </a>
          </li>
          <li
            class="cursor-pointer text-base
              
              mr-4"
          >
            <a
              href="/contact"
            >
              Contact Us
            </a>
          </li>
        </ul>
        <div
          class="sm:hidden flex flex-1 justify-end items-center"
        >
          <img
            alt="menu"
            class="w-6 h-6 object-contain mr-6"
            src="test-file-stub"
          />
          <div
            class="hidden p-6 absolute top-0 h-full right-0 flex-col border-4 border-solid bg-white"
          >
            <img
              alt="close menu"
              class="w-4 h-4 mb-6 ml-20"
              src="test-file-stub"
            />
            <ul
              class="list-none flex items-start flex-1 flex-col"
            >
              <li
                class="font-medium cursor-pointer text-base 
                  mb-4
                  border-b-2 border-solid border-black"
              >
                <a
                  href="/"
                >
                  Home
                </a>
              </li>
              <li
                class="font-medium cursor-pointer text-base 
                  mb-4
                  "
              >
                <a
                  href="/our-story"
                >
                  Our Story
                </a>
              </li>
              <li
                class="font-medium cursor-pointer text-base 
                  mb-4
                  "
              >
                <a
                  href="/rsvp"
                >
                  RSVP
                </a>
              </li>
              <li
                class="font-medium cursor-pointer text-base 
                  mb-0
                  "
              >
                <a
                  href="/contact"
                >
                  Contact Us
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="w-full flex py-4 justify-between items-center navbar border-b-4 border-solid"
    >
      <ul
        class="list-none sm:flex hidden justify-end items-center flex-1"
      >
        <li
          class="cursor-pointer text-base
              border-b-2 border-solid border-black
              mr-6"
        >
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="cursor-pointer text-base
              
              mr-6"
        >
          <a
            href="/our-story"
          >
            Our Story
          </a>
        </li>
        <li
          class="cursor-pointer text-base
              
              mr-6"
        >
          <a
            href="/rsvp"
          >
            RSVP
          </a>
        </li>
        <li
          class="cursor-pointer text-base
              
              mr-4"
        >
          <a
            href="/contact"
          >
            Contact Us
          </a>
        </li>
      </ul>
      <div
        class="sm:hidden flex flex-1 justify-end items-center"
      >
        <img
          alt="menu"
          class="w-6 h-6 object-contain mr-6"
          src="test-file-stub"
        />
        <div
          class="hidden p-6 absolute top-0 h-full right-0 flex-col border-4 border-solid bg-white"
        >
          <img
            alt="close menu"
            class="w-4 h-4 mb-6 ml-20"
            src="test-file-stub"
          />
          <ul
            class="list-none flex items-start flex-1 flex-col"
          >
            <li
              class="font-medium cursor-pointer text-base 
                  mb-4
                  border-b-2 border-solid border-black"
            >
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li
              class="font-medium cursor-pointer text-base 
                  mb-4
                  "
            >
              <a
                href="/our-story"
              >
                Our Story
              </a>
            </li>
            <li
              class="font-medium cursor-pointer text-base 
                  mb-4
                  "
            >
              <a
                href="/rsvp"
              >
                RSVP
              </a>
            </li>
            <li
              class="font-medium cursor-pointer text-base 
                  mb-0
                  "
            >
              <a
                href="/contact"
              >
                Contact Us
              </a>
            </li>
          </ul>
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
